osdep/timer.c

```diff
@@ -49,15 +49,20 @@ void mp_time_init(void)
 
 int64_t mp_time_us(void)
 {
-    int64_t r = mp_raw_time_us() - raw_time_offset;
+    return mp_time_ns() / 1000;
+}
+
+int64_t mp_time_ns(void)
+{
+    uint64_t r = mp_raw_time_ns() - raw_time_offset;
     if (r < MP_START_TIME)
         r = MP_START_TIME;
     return r;
 }
 
 double mp_time_sec(void)
 {
-    return mp_time_us() / (double)(1000 * 1000);
+    return mp_time_ns() / 1e9;
```

Comment:
is `1e9` double-typed or will this do an integer division?

Reply:
Yes, `1e9` is double literal.